src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/ProjectSteps.java

```diff
@@ -14,13 +19,194 @@
 
     @Autowired
     private Project project;
+
+    @Autowired
+    private Header header;
+
+    @Autowired
+    private ConfirmAction confirm;
+    static final String PROJECTURI = "/projects";
+
     /**
      * Create a new project.
      *
      * @param projectAttributes for create project.
      */
-    @Given("user creates a default project as")
+    @When("user creates project as")
     public void userCreatesNewProjectAs(final Map<String, String> projectAttributes) {
         project.createNewProject(projectAttributes);
     }
+
+    /**
+     * was project created with specific name.
+     */
+    @Then("validate creation on project's dashboard")
+    public void validateTheProjectIsCreatedWithSpecifyName() {
+        final String actual = project.getProjectNameOnDashboard();
+        Assert.assertEquals(actual, project.getProjectName(), "Project name match");
+    }
+
+    /**
+     * A project to delete.
+     *
+     * @param projectName the project name
+     */
+    @Given("an existing project named as {string} that user intends to delete")
+    public void anExistingProjectNamedAsThatUserIntendsToDelete(final String projectName) {
+        project.setProjectName(projectName);
+    }
+
+    /**
+     * Open context on settings.
+     */
+    @And("open project's settings")
+    public void openProjectsSettings() {
+        final String projectName = project.getProjectName();
+        project.openProjectSettingsbyName(projectName);
+    }
+
+    /**
+     * Delete project by link.
+     */
+    @When("user click over delete project link")
+    public void userClickOverDeleteProjectLink() {
+        confirm.clickOnDeleteProjectLink();
+        confirm.clickOnDeleteButton();
+    }
+
+    /**
+     * Check if project is still displayed.
+     */
+    @Then("The project no longer appear on projects section")
+    public void theProjectNoLongerAppearOnProjectsSection() {
+        Assert.assertFalse(project.isProjectListedOnMainPage(), "False if project is not listed after deletion");
+    }
+
+    /**
+     * Validate non-existance on active projects.
+     */
+    @And("the project is not present on active project")
+    public void theProjectIsNotPresentOnActiveProject() {
+        project.loadMainPage(PROJECTURI);
+        final boolean actual = project.isProjectListedOnProjectPage();
+        Assert.assertFalse(actual, "Passed if project is no longer on active project list");
+    }
+
+    /**
+     * Save on context value of project name.
+     *
+     * @param projectName the project name
+     */
+    @Given("an existing project known as {string} that user intends to edit")
+    public void anExistingProjectKnownAsThatUserIntendsToEdit(final String projectName) {
```

Comment:
what? I don't understand this step,
if this is : goes to project settings
please rename by that, if it is doing other thing improve the step name and step method name

Reply:
You are right, I was mixing two steps into one, this will be check with Carlos on Monday, to get the right steps.